app/genre.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './item'
+
+class Genre
+  attr_accessor :items
+
+  def initialize(name)
+    @id = Random.rand(1..1000)
+    @name = name
+    @items = []
+  end
```

Comment:
![image](https://user-images.githubusercontent.com/56429354/184066447-e4cd21ca-4b16-4fec-ad05-4d13583ad70b.png)

as you see from the UML diagram. items is privately attribute and name is public so you might want to do something like this 
```rb
attr_acccessor :name
attr_reader :items